### PR TITLE
Translations instructions

### PR DESCRIPTION
**Pull request by codingh2**
- Update CONTRIBUTING.md to include translation instructions.

### CODE REVIEW

**Comment by JAGFx** [comment]
Hi @codingh2

Many many thanks for your contribution!
It will be helpful. A big thank to you.

Can you update the  [Features.md](https://github.com/JAGFx/ets2-dashboard-skin/blob/master/doc/FEATURES.md#Add-a-new-language) 
(Add a translation section)  with: 

- [ ] Remove the creation part of translation file
- [ ] Add a link to your new section in **CONTRIBUTING.MD** readme.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -203,3 +203,78 @@ Example of the `maps_map_tilesVersion` entry after editing:
...
+Config: Konfigurationen
+````  
+- Save your changes after you're done.
+
+> Note: *Do not leave trailing spaces` `  and try to respect punctuation marks*.  
+
+- 4 Edit the file [src/data/config-field-values.json](https://github.com/JAGFx/ets2-dashboard-skin/blob/master/src/data/config-field-values.json) and scroll down or find the section `"general_skin_locale"`.  
+
+Append the `label` and `value` sub-section, given that **label** is the language name and **value** is the BCP 47 Code.  
+It should look like this, at the end of the section:
```
Comment (JAGFx) [inline CONTRIBUTING.md:235]: When you add something related to the configuration, you need to translate it to all other languages. In this case, the is the **German** need to be translated to French, Russian, Portuguese, etc... 

To identify the new translation key, you need to add it on the [TRANSLATION_CHANGES.md](https://github.com/JAGFx/ets2-dashboard-skin/blob/master/doc/TRANSLATION_CHANGES.md)

I've not yet created a section to identify the key to add, change or delete for the next release. 
Can you add a section named `Next release` and include the new entry on this readme?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -203,3 +203,78 @@ Example of the `maps_map_tilesVersion` entry after editing:
...
+````js
+case 'pt-PT':
+    return pt_pt;
+case 'de-DE':
+	return de_de;
+ default:
+      return {};
+````  
+
+- 6 Save every change and launch the dashboard development app with `$ npm run dashboard:dev`, as described above in **Useful commands**
```
Comment (JAGFx) [inline CONTRIBUTING.md:280]: Can you add a link to the related readme?

Comment (JAGFx) [reply CONTRIBUTING.md:280]: Can you also write something to tell the contributor to test their translation.

**Review by JAGFx — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (codingh2) [reply CONTRIBUTING.md:245]: What do you mean a section named `Next release` ? What do I put there?

**Review by JAGFx — APPROVED** [review]

**Pull request merged**